src/components/molecules/image-crop/index.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import ReactCrop, { makeAspectCrop, getPixelCrop } from 'react-image-crop'
+import classNames from 'classnames'
+import 'react-image-crop/dist/ReactCrop.css'
+
+import styles from './styles.css'
+
+const SHAPES = {
+  circle: styles.circle,
+  square: styles.square,
+}
+
+class ImageCrop extends Component {
+  state = {
+    crop: {},
+    pixelCrop: {},
+    originalImage: null,
+  }
+
+  onImageLoaded = image => {
+    let { cropShape, crop } = this.props
+    //when the shape is a circle the aspect should be 1
+    if (cropShape === 'circle') {
+      crop = { ...crop, aspect: 1 }
+    }
+    const cropProperties = makeAspectCrop(
+      {
+        ...crop,
+      },
+      image.width / image.height
```

Comment:
Is this the aspect? Will it override the `circle` one?

Reply:
Nope, onImageLoaded function when the shape is a circle the aspect will be 1